prime-router/settings/schemas/organization.json

```diff
@@ -0,0 +1,261 @@
+{
+  "$ref": "#/definitions/Organization",
+  "definitions": {
+    "Organization": {
+      "type": "object",
+      "additionalProperties": true,
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "jurisdiction": {
+          "$ref": "#/definitions/Jurisdiction",
+          "message": {
+            "required": "Jurisdiction is required for an organization"
+          }
+        },
+        "Filter": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "topic": {
+              "$ref": "#/definitions/Topic"
+            },
+            "jurisdictionalFilter": {
+              "type": ["array", "null"],
+              "items": {
+                "type": "string"
+              }
+            },
+            "qualityFilter": {
+              "type": ["array", "null"],
+              "items": {
+                "type": "string"
+              }
+            },
+            "processingModeFilter": {
+              "type": ["array", "null"],
+              "items": {
+                "type": "string"
+              }
+            },
+            "conditionFilter": {
+              "type": ["array", "null"],
+              "items": {
+                "type": "string"
+              }
+            },
+            "routingFilter": {
+              "type": ["array", "null"],
+              "items": {
+                "type": "string"
+              }
+            },
+            "mappedConditionFilter": {
```

Comment:
Is this correct?  I'm not sure this would match
```
      mappedConditionFilter:
        - type: codeString
          value: 840539006,1234
```

Reply:
seem the latest mapped condition filter changes are in master, will catch uop

Reply:
uncommented the mappedConditionFilter in local organizations.yml and re-tested, it got serialized and deserialized as expected.

Noticed that Gabrial's PR changed the name to observationFilter, but it is not in master yet, will reconcile either this PR got in first or that PR got in first...